### PR TITLE
added second booking price with commission

### PR DESCRIPTION
**Pull request by DFSko**
added second booking price with commission

resolves happy-travel/agent-app-project#114

### CODE REVIEW

File: Api/Services/Accommodations/Bookings/BookingExecution/BookingRegistrationService.cs [hunk 1]
```diff
@@ -168,6 +168,7 @@ void AddServiceDetails()
             {
                 var rate = availabilityInfo.RoomContractSet.Rate;
                 booking.TotalPrice = rate.FinalPrice.Amount;
+                booking.CreditCardPaymentPrice = rate.CreditCardPrice.Amount;
```
Comment (kirillta) [inline Api/Services/Accommodations/Bookings/BookingExecution/BookingRegistrationService.cs:171]: Ideally both prices should be in `MoneyAmount`

**Review by kirillta — APPROVED** [review]

File: Api/Services/Accommodations/Bookings/BookingExecution/BookingRegistrationService.cs (hunk 1 above)
Comment (DFSko) [reply Api/Services/Accommodations/Bookings/BookingExecution/BookingRegistrationService.cs:171]: We can't store MoneyAmount in database

File: Api/Models/Accommodations/Rate.cs [hunk 2]
```diff
@@ -45,6 +46,12 @@ namespace HappyTravel.Edo.Api.Models.Accommodations
         ///     The final and total net price of a service. This is <b>the actual</b> value of a price.
         /// </summary>
         public MoneyAmount FinalPrice { get; }
+        
+        
+        /// <summary>
```
Comment (mingazhev) [inline Api/Models/Accommodations/Rate.cs:51]: Don't think that we need a separate property for this

File: Api/Models/Accommodations/Rate.cs [hunk 3]
```diff
@@ -10,13 +10,14 @@ namespace HappyTravel.Edo.Api.Models.Accommodations
     public readonly struct Rate
     {
         [JsonConstructor]
-        public Rate(in MoneyAmount finalPrice, in MoneyAmount gross, List<Discount> discounts,
+        public Rate(in MoneyAmount finalPrice, in MoneyAmount creditCardPrice, in MoneyAmount gross, List<Discount> discounts,
```
Comment (mingazhev) [inline Api/Models/Accommodations/Rate.cs:13]: I think it will be better if final price will be a dictionary Dictionary<PaymentMethods, MoneyAmount>

**Review by mingazhev — APPROVED** [review]
Looks fine for me now, but we need to wait frontend to make changes before merge

**Review by kirillta — APPROVED** [review]

**Comment by DFSko** [comment]
not actual

**Pull request closed without merging**